tailnet/conn.go

```diff
@@ -373,6 +381,13 @@ type Conn struct {
 	watchCancel func()
 
 	trafficStats *connstats.Statistics
+	lastNetInfo  *tailcfg.NetInfo
+}
+
+func (c *Conn) GetNetInfo() *tailcfg.NetInfo {
```

Comment:
We could construct a netcheck client, but I spoke with Dean and we decided it was better to use the status of the connection being used.